packages/agent-toolkit/src/core/tools/monday-apps-tools/app-development-assistant/docs/app-development-guide-content.ts

```diff
@@ -0,0 +1,1863 @@
+const settings = JSON.parse(result.data.value);
+
+// Delete data
+await monday.storage.instance.deleteItem('user_settings');
+\`\`\`
+
+#### 4. Execute - Trigger UI Actions
+
+\`\`\`typescript
+// Open a confirmation dialog
+const confirmed = await monday.execute('confirm', {
+  message: 'Are you sure you want to delete this item?',
+  confirmButton: 'Yes, delete',
+  cancelButton: 'Cancel'
+});
+
+// Show a notice/toast
+await monday.execute('notice', {
+  message: 'Item saved successfully!',
+  type: 'success', // 'success' | 'error' | 'info'
+  timeout: 3000
+});
+
+// Open item card
+await monday.execute('openItemCard', {
+  itemId: 67890,
+  kind: 'columns' // 'columns' | 'updates'
+});
+
+// Open settings dialog
+await monday.execute('openAppFeatureModal', {
+  urlPath: '/settings',
+  urlParams: { tab: 'general' }
+});
+\`\`\`
+
+#### 5. Listen - Subscribe to Events
+
+\`\`\`typescript
+// Listen for context changes
+monday.listen('context', (res) => {
+  console.log('Context changed:', res.data);
+});
+
+// Listen for settings changes
+monday.listen('settings', (res) => {
+  console.log('Settings changed:', res.data);
+});
+
+// Listen for item IDs (for batch actions)
+monday.listen('itemIds', (res) => {
+  console.log('Selected items:', res.data);
+});
+
+// Listen for events from board view
+monday.listen('events', (res) => {
+  console.log('Event received:', res.data);
+});
+\`\`\`
+
+---`;
+
+// =============================================================================
+// MONDAY-CODE DEPLOYMENT
+// =============================================================================
+export const DEPLOYMENT_CONTENT = `## monday-code Deployment
+
+### What is monday-code?
+
+monday-code is monday.com's serverless hosting platform for app backends. It provides:
+- Automatic scaling
+- Built-in security
+- Environment variable management
+- Deployment tracking
+
+### Setting Up Backend
+
+\`\`\`bash
+# Your backend code should be in src/ directory
+# Entry point should be src/index.ts or src/index.js
+
+# Example backend structure:
+src/
+├── index.ts           # Main entry point (Express server)
+├── routes/
+│   ├── webhooks.ts    # Webhook handlers
+│   └── api.ts         # API endpoints
+├── services/
+│   └── monday.ts      # monday.com API service
+└── utils/
+    └── auth.ts        # Authentication helpers
+\`\`\`
+
+### Backend Entry Point Example
+
+\`\`\`typescript
+// src/index.ts
+import express from 'express';
+import { Router } from 'express';
+
+const app = express();
+const router = Router();
+
+app.use(express.json());
+
+// Health check endpoint (required)
+router.get('/health', (req, res) => {
+  res.status(200).json({ status: 'healthy' });
+});
+
+// Webhook endpoint example
+router.post('/webhooks/item-created', async (req, res) => {
+  const { event } = req.body;
+  
+  // Process the webhook
+  console.log('Item created:', event);
+  
+  res.status(200).json({ success: true });
+});
```

Comment:
also add here example code of using all monday code services:

- Secure storage
- Environment variables
- document DB
- queue
- other stuff?